packages/server/graphql/mutations/downgradeToStarter.ts

```diff
@@ -42,6 +60,26 @@ export default {
     }
 
     // RESOLUTION
+    if (!isSuperUser) {
+      const [billingLeaders, organization] = await Promise.all([
+        getBillingLeadersByOrgId(orgId, dataLoader),
+        dataLoader.get('organizations').load(orgId)
+      ])
+      if (!billingLeaders[0]) {
+        const error = new Error('Unable to find billing leader')
+        sendToSentry(error, {userId: viewerId})
+        return
+      }
+      analytics.organizationDowngraded(viewerId, {
```

Comment:
-1 We recording this event twice, second time it is recorded here: https://github.com/ParabolInc/parabol/blob/9ea8985892b661d4b5806deb0bcb9e06e77ae97e/packages/server/graphql/mutations/helpers/resolveDowngradeToStarter.ts#L45